aten/src/ATen/native/Dropout.cpp

```diff
@@ -0,0 +1,118 @@
+#include "ATen/ATen.h"
+#include "ATen/Dispatch.h"
+
+namespace at { namespace native {
+
+namespace {
+
+Tensor make_feature_noise(const Tensor& input) {
+  auto input_sizes = input.sizes();
+  AT_CHECK(input.dim() >= 2, "Feature dropout requires at least 2 dimensions in the input");
+  std::vector<int64_t> sizes;
+  sizes.reserve(input.dim());
```

Comment:
So wordy lol

Reply:
Yep, working with vectors is a pleasure!